WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Split Shipments/CollapsibleShipmentCard.swift

```diff
@@ -0,0 +1,72 @@
+import Yosemite
+import SwiftUI
+
+/// Displays a single collapsible shipment item row or grouped parent and child shipment item rows
+struct CollapsibleShipmentCard: View {
+    @State private var isCollapsed: Bool = true
+
+    private let viewModel: CollapsibleShipmentCardViewModel
+
+    init(viewModel: CollapsibleShipmentCardViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            mainShipmentRow
+                .padding(.horizontal, Layout.horizontalPadding)
+                .padding(.vertical, Layout.verticalPadding)
+                .background(Color(.listForeground(modal: false)))
+
+            if !isCollapsed {
+                VStack(spacing: 0) {
+                    ForEach(viewModel.childShipmentRows) { item in
+                        VStack(spacing: 0) {
+                            Divider()
+
+                            SelectableShipmentRow(viewModel: item)
+                                .padding(.leading, Layout.horizontalPadding * 2)
+                                .padding(.trailing, Layout.horizontalPadding)
+                                .padding(.vertical, Layout.verticalPadding)
```

Comment:
Should the child shipment rows have the same background color as the main shipment row? Currently there are two different row colors in dark mode: 

<img width="366" alt="image" src="https://github.com/user-attachments/assets/30471579-9d0a-42c6-8a94-8446b14bb487" />

Reply:
I missed it. Done in https://github.com/woocommerce/woocommerce-ios/pull/15397/commits/c0d7cbab4df85dfc226572e5a9161445aaad018b